.gitattributes

```diff
@@ -1 +1,2 @@
 **/snapshots/**/*.png filter=lfs diff=lfs merge=lfs -text
+**/src/androidTest/assets/*.realm filter=lfs diff=lfs merge=lfs -text
```

Comment:
Looks like the db file has been added without using git LFS, because the script `validate_lfs.sh` is complaining. Can you check please?